app/models/form_profiles/va5655.rb

```diff
@@ -1,6 +1,17 @@
 # frozen_string_literal: true
 
+module DebtManagementCenter
+  class VaAwardsComposite
+    include Virtus.model
+    attribute :name, String
+    attribute :amount, String
+    attribute :veteran_or_spouse, String
+  end
+end
```

Comment:
Any reason not to put this in it's own file? Autoloading will be problematic I think

Reply:
Done.